web/src/compose_ui.js

```diff
@@ -348,6 +348,8 @@ export function format_text($textarea, type, inserted_content) {
     const italic_syntax = "*";
     const bold_syntax = "**";
     const bold_and_italic_syntax = "***";
+    let code_syntax_start = "`";
+    let code_syntax_end = "`";
```

Comment:
I don't love this accumulate of top-level variables. For any of these variables that are only used inside a given block, like the `code` one in this case, I we should just define these variables inside that block. 

It might also be nicer to just have `const inline_code_syntax_start` and `const block_code_syntax_start` rather than using the `let` -- it's better to have if/else statements for inline vs. block in my view than avoid duplicating the call to `add_formatting` -- the inline and block cases are basically independent and it's more readable to have that be clear.

Reply:
I'll wait to test after this change is made.

Reply:
Have made the change, thanks for the suggestion!